atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServiceImpl.java

```diff
@@ -652,6 +634,94 @@ public String toString() {
         }
     }
 
+    private Map<Cell, Value> fetchAllCellsForRows(final InetSocketAddress host,
```

Comment:
This method is actually fetching the _values_ as well, not just the cells. I would name it something like `getAllCellsForRows`